src/components/Header.jsx

```diff
@@ -1,12 +1,25 @@
 import { Link } from "react-router-dom";
+import { useContext } from "react";
+import { UserContext } from "../contexts/UserContext";
 
 function Header() {
+    const { user } = useContext(UserContext)
+    
     return (
         <div>
```

Comment:
Can you think of a more semantic html element you could use instead of a `<div>`?